src/client/src/components/DraggableSidebarItem/index.tsx

```diff
@@ -116,40 +134,55 @@ const DraggableSidebarItem = ({
                   className={classnames(styles.input, {
                     [styles.azureFunctionNameInput]: isAzureFunction
                   })}
+                  maxLength={maxInputLength === undefined ? undefined : maxInputLength}
                   value={page ? page.title : azureFunctionName!.title}
                   onChange={e => {
                     if (handleInputChange && idx) {
                       handleInputChange(e.target.value, idx - 1);
                     }
                   }}
+                  onKeyDown={handleKeyDownInput}
                 />
               ) : (
-                <input
-                  className={classnames(
-                    styles.disabledInput,
-                    styles.input,
-                    customInputStyle
-                  )}
-                  value={text}
-                  disabled={true}
-                />
-              )}
+                  <input
+                    className={classnames(
+                      styles.disabledInput,
+                      styles.input,
+                      customInputStyle
+                    )}
+                    value={text}
+                    disabled={true}
+                  />
+                )}
             </div>
           </div>
+          {pageNameMaxLength && (<div
+            className={classnames({
+              [styles.errorTextContainer]:
+                withIndent || reorderSvgUrl || true,
```

Comment:
Always true?

Reply:
Seems like this code was first checked in here #509 and copied over, but ya prob should take a look at this

Reply:
it's not a truth conditional, its an existence one. `reorderSvgUrl` is a string so if it is undefined, then the style is true, else it will inherit the value of reorderSvgUrl. `true` is a default value in a sense